### ISSUE TITLE
Please update the PDF links at https://www.onrr.gov/Laws_R_D/FRNotices/AA15.htm

### ISSUE THREAD

**Issue by cthomasONRR** [ONRR/onrr.gov-site#396]
Hello,

In an effort to make the following website more 508-Compliant, we need to switch out the PDF links with links that instead point to the Federal Register website.

On webpage: https://www.onrr.gov/Laws_R_D/FRNotices/AA15.htm,

Please update/replace the "Amendments to Designated Areas: (81 FR 49897 -- July 29, 2016)" PDF link ([https://www.onrr.gov/Laws_R_D/FRNotices/PDFDocs/49897.pdf](url)) with the following link: 
[https://www.federalregister.gov/documents/2016/07/29/2016-17599/amendments-to-designated-areas](url)

Please update/replace the "Final Rulemaking:  (80 FR 24794 --May 1, 2015)" PDF link ([https://www.onrr.gov/Laws_R_D/FRNotices/PDFDocs/24794.pdf](url)) with the following link:
[https://www.federalregister.gov/documents/2015/05/01/2015-09955/indian-oil-valuation-amendments](url)

Please update/replace the "Proposed Rulemakings:  (79 FR 35102--June 19, 2014)" PDF link ([https://www.onrr.gov/Laws_R_D/FRNotices/PDFDocs/35102.pdf](url)) with the following link:
[https://www.federalregister.gov/documents/2014/06/19/2014-13967/indian-oil-valuation-amendments](url)

Please let me know if you have any questions.

Thanks,
Christine

**Comment by lpgoldstein** [comment]
@cthomasONRR Thanks! Please check and see if this looks okay before we go live: http://2017onrr/Laws_R_D/FRNotices/AA15.htm

**Comment by cthomasONRR** [comment]
@lpgoldstein, I just checked and the new links work for me.
I approve these changes to go live.

Thanks!

**Comment by lpgoldstein** [comment]
It should be live! @cthomasONRR
